<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitReportReporter -->
<testsuite skipped="2" hostname="Lucho-PC" name="sumat.test.GenerarPlanillaDeAutoliquidacionTest" tests="3" failures="0" timestamp="09 ene 2020 12:48:40 GMT" time="12.950" errors="1">
  <testcase name="simple" time="12.950" classname="sumat.test.GenerarPlanillaDeAutoliquidacionTest">
    <error type="java.lang.NullPointerException">
      <![CDATA[java.lang.NullPointerException
at sumat.pages.GenerarPlanillaDeAutoliquidacionPage.validar_mensaje(GenerarPlanillaDeAutoliquidacionPage.java:32)
at sumat.pages.GenerarPlanillaDeAutoliquidacionPage.RET(GenerarPlanillaDeAutoliquidacionPage.java:56)
at sumat.test.GenerarPlanillaDeAutoliquidacionTest.simple(GenerarPlanillaDeAutoliquidacionTest.java:83)
at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
at java.lang.reflect.Method.invoke(Unknown Source)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:124)
at org.testng.internal.Invoker.invokeMethod(Invoker.java:583)
at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:719)
at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:989)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
at org.testng.TestRunner.privateRun(TestRunner.java:648)
at org.testng.TestRunner.run(TestRunner.java:505)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:455)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:450)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:415)
at org.testng.SuiteRunner.run(SuiteRunner.java:364)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:84)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1208)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1137)
at org.testng.TestNG.runSuites(TestNG.java:1049)
at org.testng.TestNG.run(TestNG.java:1017)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
    </error>
  </testcase> <!-- simple -->
  <testcase name="test" classname="sumat.test.GenerarPlanillaDeAutoliquidacionTest">
    <skipped/>
  </testcase> <!-- test -->
  <testcase name="dbtest" classname="sumat.test.GenerarPlanillaDeAutoliquidacionTest">
    <skipped/>
  </testcase> <!-- dbtest -->
</testsuite> <!-- sumat.test.GenerarPlanillaDeAutoliquidacionTest -->
